src/test/rgw/test_rgw_amqp.cc

```diff
@@ -61,7 +61,7 @@ class TestAMQP : public ::testing::Test {
 TEST_F(TestAMQP, ConnectionOK)
 {
   const auto connection_number = amqp::get_connection_count();
-  conn = amqp::connect("amqp://localhost", "ex1", false);
+  conn = amqp::connect("amqp://localhost", "ex1", false, false, false, boost::none);
```

Comment:
see: https://github.com/ceph/ceph/pull/39392/files#r573599891
we can probably add a simple test that verify the ssl socket was created based on the URL